src/installing-workflow/configuring-object-storage.md

```diff
@@ -32,6 +32,7 @@ The helm classic chart for Deis Workflow can be easily configured to connect Wor
             ```
             export AWS_ACCESS_KEY, AWS_SECRET_KEY, AWS_REGISTRY_BUCKET, AWS_DATABASE_BUCKET, AWS_BUILDER_BUCKET, S3_REGION
             ```
+              * If you want to use [IAM credentials][aws-iam] you need not set AWS_ACCESS_KEY and AWS_SECRET_KEY.
```

Comment:
I just tried to write similar to how the rest of this doc does to maintain consistency. I am ok to change if needed.